package/MDAnalysis/lib/distances.py

```diff
@@ -101,6 +105,8 @@
    :class:`~MDAnalysis.core.groups.AtomGroup` or an :class:`np.ndarray`
 .. versionchanged:: 2.5.0
    Interface to the `distopia`_ package added.
+.. versionchanged:: 2.7.0
+   Distopia support greatly expanded (with distopia ≥ 0.3.0).
```

Comment:
We do need 0.3.1 because this code cannot detected 0.3.0:
```suggestion
   Distopia support greatly expanded (with distopia ≥ 0.3.1).
```